tests/regression/tests/RESPONSE-955-WEB-SHELLS/955260.yaml

```diff
@@ -0,0 +1,30 @@
+---
+meta:
+  author: "azurit"
+  enabled: false
+  name: "955260.yaml"
+  description: "Regression tests for rule 955260"
+tests:
+  - test_title: 955260-1
+    desc: "Matching web shell Ru24PostWebShell. Our test infrastructure, currently, cannot run this test properly because of multiline output, so it's disabled."
+    stages:
+      - stage:
+          input:
+            dest_addr: "127.0.0.1"
+            port: 80
+            headers:
+              Host: "localhost"
+              User-Agent: "OWASP CRS test agent"
+              Accept: "text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5"
+              Accept-Encoding: "gzip,deflate"
+              Accept-Language: "en-us,en;q=0.5"
+              Content-Type: "text/plain"
+            method: "GET"
+            version: "HTTP/1.0"
+            uri: "/anything"
```

Comment:
As per our [documentation](http://localhost:1313/documentation/development/contribution_guidelines/#using-the-correct-http-endpoint) the **uri** should be "/get".

Reply:
Fixed.